copyright_and_licenses.csv

```diff
@@ -147,6 +147,8 @@ fr_ign_RAMG2016.tif,Institut National Geographique (IGN) France,Open License Fra
 fr_ign_RAR07_bl.tif,Institut National Geographique (IGN) France,Open License France - https://www.etalab.gouv.fr/wp-content/uploads/2014/05/Open_Licence.pdf,,
 fr_ign_RASPM2018.tif,Institut National Geographique (IGN) France,Open License France - https://www.etalab.gouv.fr/wp-content/uploads/2014/05/Open_Licence.pdf,,
 fr_ign_README.txt,Disclaimed,Public domain,,
+hu_bme_geod_etrs2eov_notowgs.tif,Budapest University of Technology and Economics - Faculty of Civil Engineering,CC-BY-4.0,1.8,
+hu_bme_geod_geoid_eht2014.tif,Budapest University of Technology and Economics - Faculty of Civil Engineering,CC-BY-4.0,1.8,
```

Comment:
Please also add a line for hu_bme_geod_README.txt below that one

Reply:
Done.